builder-base/scripts/install_base_yum_packages.sh

```diff
@@ -19,29 +19,32 @@ set -o pipefail
 SCRIPT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
 source $SCRIPT_ROOT/common_vars.sh
 
+BASE_DIRECTORY="$(git rev-parse --show-toplevel)"
```

Comment:
Id probably skip this file since youll need to change the dockerfile for all the layers to include the new common.sh file.